pkg/reconciler/service/resources/configuration_test.go

```diff
@@ -127,3 +129,14 @@ func TestInlineConfigurationSpec(t *testing.T) {
 		t.Errorf("expected %q labels got %q", want, got)
 	}
 }
+
+func TestConfigurationHasNoKubectlAnnotation(t *testing.T) {
+	s := createServiceWithKubectlAnnotation()
+	c, err := makeConfiguration(s)
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
```

Comment:
Still not fatal here. And below.

Reply:
Sorry, I missed that. Also fixed the other occurrences of 

```
        if err != nil {
              t.Errorf("Unexpected error: %v", err)
        }
```

in this file.